src/screens/DiscoveryScreenV2/elements/RecentSearchItem.js

```diff
@@ -1,55 +1,51 @@
 import * as React from 'react';
-import {Pressable, StyleSheet, Text} from 'react-native';
+import { Pressable, StyleSheet, Text } from 'react-native';
 
 import IcArrowUpRight from '../../../assets/icons/ic_arrow_up_right';
-import {fonts} from '../../../utils/fonts';
-import {COLORS} from '../../../utils/theme';
+import { colors } from '../../../utils/colors';
+import { fonts } from '../../../utils/fonts';
 
 /**
  * @typedef {Object} RecentSearchItemsProp
  * @property {String} text
  * @property {Function} onItemClicked
  */
 /**
- *
+ * 
  * @param {RecentSearchItemsProp} prop
  */
 const RecentSearchItems = (prop) => {
-  const {onItemClicked, text = 'Coba'} = prop;
-  return (
-    <Pressable
-      style={styles.pressableContainer}
-      onPress={onItemClicked}
-      android_ripple={{
-        color: COLORS.gray9,
-        borderless: false
-      }}>
-      <Text style={styles.text}>{text}</Text>
-      <IcArrowUpRight style={styles.icon} />
+    const { onItemClicked, text = "Coba" } = prop
+    return <Pressable style={styles.pressableContainer} onPress={onItemClicked}
+        android_ripple={{
+            color: colors.gray1,
+            borderless: false,
+        }}>
+        <Text style={styles.text}>{text}</Text>
+        <IcArrowUpRight style={styles.icon} />
     </Pressable>
-  );
-};
+}
 
 const styles = StyleSheet.create({
-  icon: {
-    alignSelf: 'center'
-  },
-  pressableContainer: {
-    paddingLeft: 20,
-    paddingRight: 14,
-    display: 'flex',
-    flexDirection: 'row',
-    width: '100%'
-  },
-  text: {
-    fontFamily: fonts.inter[500],
-    fontSize: 14,
-    lineHeight: 17,
-    flex: 1,
-    alignSelf: 'center',
-    paddingTop: 6,
-    paddingBottom: 6
-  }
-});
+    icon: {
+        alignSelf: 'center',
+    },
+    pressableContainer: {
+        paddingLeft: 20,
+        paddingRight: 14,
+        display: 'flex',
+        flexDirection: 'row',
+        width: '100%',
+    },
+    text: {
+        fontFamily: fonts.inter[500],
+        fontSize: 14,
+        lineHeight: 17,
+        flex: 1,
+        alignSelf: 'center',
+        paddingTop: 6,
+        paddingBottom: 6,
+    }
+})
 
-export default RecentSearchItems;
+export default RecentSearchItems
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There should be a newline at the end of the file. This is a common convention that helps prevent certain issues when concatenating or processing files.
```diff
export default RecentSearchItems
+52: 
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->